src/index.js

```diff
@@ -49,7 +49,7 @@ const decorator = ({ declareQueries, declareCommands, declareConnect }) => (Comp
     loadingDecorator = noLoaderLoading, // force a "safety" loader
     reduceQueryProps: reduceQueryPropsFn,
     mapProps,
-    __DO_NOT_USE_additionalPropTypes: __props
+    propTypes: __props
```

Comment:
well, kind of.. all the other props are derived and you can't avoid them. These are the only possible user-defined ones, so I think it makes sense to spare the extra word